### ISSUE TITLE
Heywood Healthcare not scraping

### ISSUE THREAD

**Issue by harcod** [livgust/covid-vaccine-scrapers#215]
I received a notice from another site of availability at Heywood Healthcare this morning at 4/5/21 10:03am.
> 750 appointments available at Heywood Healthcare in Gardner, MA on 04/08/2021. Check eligibility and sign up at https://t.co/vD7sxHbs7W

We reported 0.
![image](https://user-images.githubusercontent.com/546007/113583549-c33dcc80-95f7-11eb-91ba-a221f9059c80.png)

**Comment by kriation** [comment]
@harcod - Do we have HTML or an image capture of the site when the appointments were available?

**Comment by harcod** [comment]
@kriation - https://macovidvaccines.harcod.com/html/gardnervaccinations.as.me-schedule.php.html

**Comment by pfurbacher** [comment]
I don't know if @kriation is working on this but just out of curiosity, I did some digging into Heywood HC, and found it's another Acuity Scheduling powered site, but an interesting variant. I was able to borrow a good bit of code from the Greater Lawrence FHC scraper (non-browser approach), got the "More Times" page advancing mechanism working, and was pulling in bucket loads of slots this afternoon and evening. The code is on my fork, in the HeywoodHealthcare branch.
